core/serial_command.py

```diff
@@ -362,30 +315,30 @@ def fire_continuous(self):
         self.disable_external_trigger()
         if self._firing is True:
             raise tellie_exception.TellieException("Cannot fire, already in firing mode")
-        self._send_command(_cmd_fire_continuous, False)
+        self._send_command(p._cmd_fire_continuous, False)
         self._firing = True
         self._force_setting = False
 
-    def read_buffer(self, n=100):
+    def read_buffer(self, n=p._read_bytes):
         return self._serial.read(n)
```

Comment:
Done, replaced all other calls to _serial_read(n) with read_buffer(), i.e. using the default value p._read_bytes